pubspec.yaml

```diff
@@ -4,43 +4,44 @@ version: 0.16.0
 homepage: https://github.com/dart-lang/pana
 
 environment:
-  sdk: '>=2.10.0 <3.0.0'
+  sdk: ">=2.10.0 <3.0.0"
 
 dependencies:
-  analyzer: '>=0.41.0 <2.0.0'
-  args: '>=1.6.0 <3.0.0'
-  async: '>=1.13.3 <3.0.0'
-  cli_util: '>=0.2.0 <0.4.0'
-  html: '>=0.13.3 <0.16.0'
-  http: '>=0.11.3 <0.14.0'
-  io: '>=0.3.4 <2.0.0'
-  json_annotation: '>=2.0.0 <5.0.0'
-  logging: '>=0.11.3+1 <2.0.0'
-  markdown: '>=3.0.0 <5.0.0'
+  analyzer: ">=0.41.0 <2.0.0"
```

Comment:
let's avoid formatting changes here

Reply:
oops yeah, sorry that automatically happens and goes without my knowledge, I'll revert it back.